Documentation/concepts/overview.rst

```diff
@@ -11,97 +11,81 @@ Component Overview
-  queries identity metadata like container / pod labels, which are used to
-  identify `endpoints` in Cilium security policies.
-
-* Interacts with the container platforms network plugin to perform IP address
-  management (IPAM), which controls what IPv4 and IPv6 addresses are assigned
-  to each container. The IPAM is managed by the agent in a shared pool between
-  all plugins which means that the Docker and CNI network plugin can run side
-  by side allocating a single address pool.
-
-* Combines its knowledge about container identity and addresses with the
-  already configured security and visibility policies to generate highly
-  efficient BPF programs that are tailored to the network forwarding and
-  security behavior appropriate for each container.
-
-* Compiles the BPF programs to bytecode using `clang/LLVM
-  <https://clang.llvm.org/>`_ and passes them to the Linux kernel to run for
-  all packets in / out of the container's virtual ethernet device(s).
-
-
-Cilium CLI Client
-=================
-
-The Cilium CLI Client (cilium) is a command-line tool that is installed along
-with the Cilium Agent.  It gives a command-line interface to interact with all
-aspects of the Cilium Agent API.   This includes inspecting Cilium's state
-about each network endpoint (i.e., container), configuring and viewing security
-policies, and configuring network monitoring behavior.
-
-Linux Kernel BPF
-================
-
-Berkeley Packet Filter (BPF) is a Linux kernel bytecode interpreter originally
-introduced to filter network packets, e.g. tcpdump and socket filters. It has
-since been extended with additional data structures such as hashtable and
-arrays as well as additional actions to support packet mangling, forwarding,
-encapsulation, etc. An in-kernel verifier ensures that BPF programs are safe to
-run and a JIT compiler converts the bytecode to CPU architecture specific
-instructions for native execution efficiency. BPF programs can be run at
-various hooking points in the kernel such as for incoming packets, outgoing
-packets, system calls, kprobes, etc.
+A deployment of Cilium and Hubble consists of the following components running
+in a cluster:
+
+
+Cilium
+======
+
+Agent
+  The Cilium agent (``cilium-agent``) runs on each node in the cluster. At a
+  high-level, the agent accepts configuration via Kubernetes or APIs that
+  describes networking, service load-balancing, network policies, and
+  visibility & monitoring requirements.
+
+  The Cilium agent listens for events from orchestration systems such as
+  Kubernetes to learn when containers or workloads are started or stopped. It
+  manages the eBPF programs which the Linux kernel uses to control all network
+  access in / out of those containers.
+
+Client (CLI)
+  The Cilium CLI client (``cilium``) is a command-line tool that is installed
+  along with the Cilium Agent. It interacts with the REST API of the Cilium
```

Comment:
The agent has been referred to by its lower-case counterpart several times in this paragraph and the paragraph before it. I'd suggest we stick to one capitalization or the other.